db/maintenance/list.go

```diff
@@ -0,0 +1,56 @@
+package maintenance
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/olekukonko/tablewriter"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/cli/utils"
+	"github.com/Scalingo/go-scalingo/v6"
+)
+
+func List(ctx context.Context, app string, addonName string, paginationOpts scalingo.PaginationOpts) error {
+	caser := cases.Title(language.English)
```

Comment:
Out of curiosity why are you using `cases`?

Reply:
Because strings.Title is deprecated: https://stackoverflow.com/questions/71620717/in-go-1-18-strings-title-is-deprecated-what-to-use-now-and-how